.github/dependabot.yml

```diff
@@ -8,25 +8,4 @@ updates:
       open-pull-requests-limit: 1
       target-branch: 'dev'
       labels:
-          - 'external-dependency'
-
-    - package-ecosystem: 'npm'
-      directory: '/branding'
-      schedule:
-          interval: 'monthly'
-          day: 'monday'
-      open-pull-requests-limit: 1
-      target-branch: 'dev'
-      labels:
-          - 'external-dependency'
-
-    - package-ecosystem: 'npm'
-      directory: '/ui'
-      schedule:
-          interval: 'monthly'
-          day: 'monday'
-      open-pull-requests-limit: 1
-      target-branch: 'dev'
-      labels:
-          - 'external-dependency'
-          
\ No newline at end of file
+          - 'external-dependency'
```

Comment:
It is there, I deleted branding folder

Reply:
Got it, I wasn't seeing the entire file in the diff